asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -155,17 +144,12 @@ def get_html_footer(self):
         """
         Get HTML footer for pose visualization
         """
-        if self.target == "sars2_mpro":
-            return colour_sars2_mpro + orient_tail_sars2_mpro
-        elif self.target == "mers_mpro":
-            return colour_mers_mpro + orient_tail_mers_mpro
-        elif self.target == "7ene_mpro":
-            return colour_7ene_mpro + orient_tail_7ene_mpro
-        elif self.target == "272_mpro":
-            return colour_mers_mpro + orient_tail_272_mpro
-        elif self.target == "sars2_mac1":
-            return colour_sars2_mac1 + orient_tail_sars2_mac1
-        else:
-            raise ValueError(
-                f"Target {self.target} does not have an HTML visualiser element implemented."
-            )
+        # colour uses target name
+        protein_name = VizTargets.get_target_name(self.target, underscore=True)
+        colour = getattr(HTMLBlockData, f"colour_{protein_name}")
```

Comment:
two functions on the `HTMLBlockData` might work better here one something like `get_pocket_colour` and `get_orient_tail` which just takes the targets name and handles all of the underscore matchings internally might be easier to follow and allow you to reuse a single matching logic.